cdp_client/cdp.py

```diff
@@ -218,16 +218,19 @@ def for_each_child(self, callback):
     def subscribe_to_structure_changes(self, callback):
         self._structure_subscriptions.append(callback)
 
-    def subscribe_to_value_changes(self, callback):
-        self._value_subscriptions.append(callback)
-        self._connection.send_value_request(self._id())
+    def subscribe_to_value_changes(self, callback, fs=5):
+        self._value_subscriptions.append((callback, fs))
+        self._connection.send_value_request(self._id(), fs)
 
     def unsubscribe_from_structure_changes(self, callback):
         self._structure_subscriptions.remove(callback)
 
     def unsubscribe_from_value_changes(self, callback):
-        self._value_subscriptions.remove(callback)
-        if not self._value_subscriptions:
+        self._value_subscriptions = [i for i in self._value_subscriptions if i[0] != callback]
+        if self._value_subscriptions:
+            max_fs = max(self._value_subscriptions, key=lambda e: e[1])
```

Comment:
Should have a test to test that the value_send_request with new fs (existing max fs) will be sent correctly on unsubscribe.

Reply:
added